src/pages/Home/Home.jsx

```diff
@@ -64,6 +65,27 @@ export default function Home() {
         setError(true);
         // setResult(`Shipment with tracking ID: ${tracking_id} not found!`);
       }
+      console.log(options)
+
+      const options2 = {
+        method: 'POST',
+        url: 'http://localhost:5000/api/loc-history-id',
```

Comment:
```suggestion
        url: 'http://postoffice-api.herokuapp.com/api/loc-history-id',
```

Don't use localhost for finished code, max can always do a deploy of the backend if it's missing the endpoint.